tests/test_compose.py

```diff
@@ -533,6 +533,20 @@ def test_searchpath_config_errors(
         with expected:
             compose(config_name=config_name, overrides=overrides)
 
+    def test_searchpath_invalid(
+        self,
+        init_configs: Any,
+    ) -> None:
+        config_name = "without_sp"
+        override = "hydra.searchpath=['pkg://fakeconf']"
+        with warns(
+            expected_warning=UserWarning,
+            match=re.escape(
+                "provider=hydra.searchpath in command-line, path=fakeconf is not available."
+            ),
+        ):
+            compose(config_name=config_name, overrides=[override])
```

Comment:
no need for a dedicated config here. you can test it with an existing app:
```
 python  examples/tutorials/basic/your_first_hydra_app/1_simple_cli/my_app.py hydra.searchpath=['pkg://fakeconf']
```

Reply:
`without_sp` is not a new config, I've removed the extra config I added previously.